src/constants/pinning.js

```diff
@@ -32,6 +32,12 @@ const pinningServiceTemplates = [
     apiEndpoint: 'https://api.filebase.io/v1/ipfs',
     visitServiceUrl: 'https://docs.filebase.com/api-documentation/ipfs-pinning-service-api'
   },
+  {
+    name: 'Functionland',
+    icon: 'https://bafybeidgnnvgm6i3pfzhjakgzcdwdbvwtmkkx7vzeiyvtk2b4oxis53vhm.ipfs.dweb.link/',
+    apiEndpoint: 'https://api.cloud.fx.land',
+    visitServiceUrl: 'https://docs.fx.land/pinning-service/ipfs-pinning-service-api'
```

Comment:
@ehsan6sha what is the ETA for general availability?

The docs at https://docs.fx.land/pinning-service/ipfs-pinning-service-api#authentication state it is invite-only, and https://cloud.fx.land/ is a broken link.

WebUI is shipping inside IPFS Desktop, and as bare minimum for reviewing this PR, a regular user needs to be able to use it.

Reply:
Thank you @lidel for mentioning that . We were planning to officially announce the release later, but I made it live now. the website and instructions are updated and easy to use for anyone:
https://cloud.fx.land/